packages/store/src/operators/of-action.ts

```diff
@@ -74,6 +74,40 @@ export function ofActionErrored(...allowedTypes: any[]) {
   return ofActionOperator(allowedTypes, [ActionStatus.Errored]);
 }
 
+function mapToActionExecuting(allowedType: any) {
+  return (source: Observable<ActionContext>) =>
+    merge(
+      source.pipe(
+        ofActionDispatched(allowedType),
+        mapTo(true)
+      ),
+      source.pipe(
+        ofActionCompleted(allowedType),
+        mapTo(false)
+      )
+    );
+}
+
+/**
+ * RxJS operator for selecting out specific actions.
+ *
+ * This will grab an action and return true when dispatched and false when completed
+ * When used with multiple actions, will return true when ALL actions has been dispatched or only some of them has completed,
+ * and false when ALL are completed
+ */
+export function ofActionExecuting(...allowedTypes: any) {
+  return (o: Observable<ActionContext>) =>
+    combineLatest(...allowedTypes.map((type: any) => o.pipe(mapToActionExecuting(type)))).pipe(
```

Comment:
```ts

return (stream: Observable<ActionContext>) => {
  //  ?  - what type
  const executionTypes: Array<Observable< ? >> = allowedTypes.map(
      (type: ActionType) => stream.pipe(mapToActionExecuting(type))
  );
  
  return combineLatest(...executionTypes).pipe(
    map((actionsExecuting: boolean[]) => {
        const score: number = actionsExecuting
          .reduce((score: number, status: boolean) => score + Number(status), 0);

        return score > 0 ? score === actionsExecuting.length : false;
    })
  );
});

```

Reply:
just score > 0 does the trick, since with at least one true, we consider it "executing"